### PR TITLE
test shift_period_suffix

### PR DESCRIPTION
**Pull request by aarek**
LOGGER: Add shift_period_suffix option

References #473

### CODE REVIEW

File: library/logger/logger/new_spec.rb [hunk 1]
```diff
@@ -60,4 +60,26 @@
...
+
+      logger.add Logger::INFO, 'message'
+
+      Time.stub!(:now).and_return(tomorrow)
+      logger.add Logger::INFO, 'second message'
+
+      shifted_path = "#{path}.#{now.strftime(shift_period_suffix)}"
+
+      File.exist?(shifted_path).should == true
+
```
Comment (eregon) [inline library/logger/logger/new_spec.rb]: I think we need `logger.close` here, to avoid the error on AppVeyor.

**Comment by eregon** [comment]
Could you rebase this PR on top of latest `master`?

**Comment by aarek** [comment]
@eregon done

**Comment by eregon** [comment]
Thank you for the new spec!

**Pull request merged**